pkg/route/api/helper.go

```diff
@@ -0,0 +1,16 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+// IngressConditionStatus returns the first status and condition matching the provided ingress condition type.
```

Comment:
Should not have duplicates, clients are allowed to take first matching
condition and assume it is correct